tutorials/step_by_step/singletons_autoload.rst

```diff
@@ -56,49 +54,49 @@ access it by requesting:
 Custom scene switcher
 ---------------------
 
-This short tutorial will explain how to make a scene switcher by using
+This short tutorial will explain how to make a scene switcher using
 autoload. For simple scene switching, the
 :ref:`SceneTree.change_scene() <class_SceneTree_change_scene>`
 method suffices (described in :ref:`doc_scene_tree`), so this method is for
-more complex behaviors when switching scenes.
+more complex behavior when switching between scenes.
 
 First download the template from here:
 :download:`autoload.zip </files/autoload.zip>`, then open it.
 
 Two scenes are present, scene_a.scn and scene_b.scn on an otherwise
 empty project. Each are identical and contain a button connected to a
-callback for going to the opposite scene. When the project runs, it
-starts in scene_a.scn. However, this does nothing and pressing the
+callback for switching to the other scene. When the project runs, it
+starts in scene_a.scn. However, this currently does nothing and pressing the
 button does not work.
 
 global.gd
 ---------
 
-First of all, create a global.gd script. The easier way to create a
+First of all, create a global.gd script. The easy way to create a
 resource from scratch is from the resources tab:
 
 .. image:: /img/newscript.png
 
-Save the script to a file global.gd:
+Save the script as `global.gd`:
 
 .. image:: /img/saveasscript.png
 
-The script should be opened in the script editor. Next step will be
-adding it to autoload, for this, go to: Scene [STRIKEOUT:> Project
-Settings]> AutoLoad and add a new autoload with name "global" that
-points to this file:
+The script should open in the script editor. The next step is to add
+it to AutoLoad list. Select Scene [STRIKEOUT:> Project
```

Comment:
The `[STRIKEOUT:> Project Settings]` is a bug from the parser used to convert the documentation to reST, so it should be fixed actually :) It should be "Select `Scene > Project Settings` from the menu" instead.